samples/nfc/record_launch_app/README.rst

```diff
@@ -0,0 +1,75 @@
+.. _record_launch_app:
+
+NFC: Launch App
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The NFC Launch App sample shows how to use the NFC tag to launch an app on the polling smartphone.
+It uses the :ref:`lib_nfc_ndef` library.
+
+Requirements
+************
+
+This sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840,  nrf52dk_nrf52832, nrf52833dk_nrf52833
+
+
+The sample also requires a smartphone or a tablet.
+You need to have the nRF Toolbox app installed for iOS devices.
+
+Overview
+********
+
+When the sample starts, it initializes the NFC tag and generates an NDEF message with two records that contain the app Universal Link and Android Application Record.
+Then it sets up the NFC library to use the generated message and sense the external NFC field.
+
+The only events handled by the application are the NFC events.
+
+User interface
+**************
+
+LED 1:
+   Indicates if an NFC field is present.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nfc/record_launch_app`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, perform the following steps to test it:
+
+1. Touch the NFC antenna with the smartphone or tablet and observe that **LED 1** is lit.
+#. Observe that the smartphone or tablet launches the nRF Toolbox application.
```

Comment:
Shall we clarify the smartphone either launches the application or opens an appstore with the app?

Reply:
I will add this information now.